### DISCUSSION TITLE
SageTV Equipment for Sale

### DISCUSSION THREAD

**Discussion by gregort3** [OpenSageTV/Community#23] [Marketplace]
I have 3 x STP-HD300 available. This includes 3 power supplies but only 3 remotes. Good working order.  Selling from Houston Tx . Shipping not included & buyer pays shipping. Please message me if interested and we can figure out price & shipping.

**Comment by weston1968** [comment]
I would like 2

**Reply by gregort3** [reply]
Yes I do have 2 of these to Sell
Thinking  $30 Each Includes Power Cable and Remote Control , I tested them to today to make sure they are still working . 
HD Theater 300 Model STP-HD300
I was a SageTV User for many years. 
Do you have  any Questions 

Shipping not included & buyer pays shipping.
I would Estimate Shipping around $12 to $15 , I would have to find out how much if you out of Country or out of 48 US States.

Not sure about Payment from you side. 1st time selling from git hub.

Thanks

Gregort

**Reply by gregort3** [reply]
PayPal I will need to setup a Account Let you know
Question why would you not need Power Supply  or Remote Control, the Sage Box ?
What use would the box be without these. I still will want to sell the HD Theater 300.
Thanks

**Comment by eelpout** [comment]
I'll take the third if it's still available.

**Comment by gregort3** [comment]
Yes I do have 1 of these to Sell
Thinking $30  Includes Power Cable and Remote Control , I tested today to make sure they are still working .
HD Theater 300 Model STP-HD300
I was a SageTV User for many years.
Do you have any Questions

Shipping not included & buyer pays shipping.
I would Estimate Shipping around $12 to $15 , I would have to find out how much if you out of Country or out of 48 US States.

Not sure about Payment from you side. 1st time selling from git hub.

**Reply by eelpout** [reply]
i'd still take the one if it's still available. $50 for item & shipping?

do you have a pay pal email? (Probably putting it on here with space between the letters is best)

**Comment by weston1968** [comment]
I will take all three.  Can I paypal you?

On Mon, Jul 24, 2023 at 6:47 PM gregort3 ***@***.***> wrote:

> Yes I do have 2 of these to Sell
> Thinking $30 Each Includes Power Cable and Remote Control , I tested them
> to today to make sure they are still working .
> HD Theater 300 Model STP-HD300
> I was a SageTV User for many years.
> Do you have any Questions
>
> Shipping not included & buyer pays shipping.
> I would Estimate Shipping around $12 to $15 , I would have to find out how
> much if you out of Country or out of 48 US States.
>
> Not sure about Payment from you side. 1st time selling from git hub.
>
> Thanks
>
> Gregort
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6534344>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAAPLSL6B5FPQBBLGXTXR33OHANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
Lance Weston
9 Claudine Court
East Northport, NY
11731
6319024700

On Mon, Jul 24, 2023 at 6:47 PM gregort3 ***@***.***> wrote:

> Yes I do have 2 of these to Sell
> Thinking $30 Each Includes Power Cable and Remote Control , I tested them
> to today to make sure they are still working .
> HD Theater 300 Model STP-HD300
> I was a SageTV User for many years.
> Do you have any Questions
>
> Shipping not included & buyer pays shipping.
> I would Estimate Shipping around $12 to $15 , I would have to find out how
> much if you out of Country or out of 48 US States.
>
> Not sure about Payment from you side. 1st time selling from git hub.
>
> Thanks
>
> Gregort
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6534344>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAAPLSL6B5FPQBBLGXTXR33OHANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
Sorry, did not see someone else wanted on.  Two is fine

Lance

On Mon, Jul 24, 2023 at 6:48 PM gregort3 ***@***.***> wrote:

> Yes I do have 1 of these to Sell
> Thinking $30 Includes Power Cable and Remote Control , I tested today to
> make sure they are still working .
> HD Theater 300 Model STP-HD300
> I was a SageTV User for many years.
> Do you have any Questions
>
> Shipping not included & buyer pays shipping.
> I would Estimate Shipping around $12 to $15 , I would have to find out how
> much if you out of Country or out of 48 US States.
>
> Not sure about Payment from you side. 1st time selling from git hub.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6534353>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAGEUZOCZRFUVEY22ULXR33TXANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
$30 is too low.  I think $50 is more fair.  So I will pay $50 each +
shipping.  Do not need remotes or power cables.

Lance

On Mon, Jul 24, 2023 at 6:47 PM gregort3 ***@***.***> wrote:

> Yes I do have 2 of these to Sell
> Thinking $30 Each Includes Power Cable and Remote Control , I tested them
> to today to make sure they are still working .
> HD Theater 300 Model STP-HD300
> I was a SageTV User for many years.
> Do you have any Questions
>
> Shipping not included & buyer pays shipping.
> I would Estimate Shipping around $12 to $15 , I would have to find out how
> much if you out of Country or out of 48 US States.
>
> Not sure about Payment from you side. 1st time selling from git hub.
>
> Thanks
>
> Gregort
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6534344>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAAPLSL6B5FPQBBLGXTXR33OHANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
I can send you a check form my on line account.  I just want the HD300's as
spares.  I use universal remotes and the SageTV remotes are of no use to
me.  Give them to the guy who only wants one HD300.

If you give me your name and address I will get a check for $125 right out
to you.

Lance

On Tue, Jul 25, 2023 at 11:50 AM gregort3 ***@***.***> wrote:

> PayPal I will need to setup a Account Let you know
> Question why would you not need Power Supply or Remote Control, the Sage
> Box ?
> What use would the box be without these. I still will want to sell the HD
> Theater 300.
> Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6541999>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAFYXPGVH4R7QRULMY3XR7TMJANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
Hi,

What is the status?

Lance

On Tue, Jul 25, 2023 at 11:50 AM gregort3 ***@***.***> wrote:

> PayPal I will need to setup a Account Let you know
> Question why would you not need Power Supply or Remote Control, the Sage
> Box ?
> What use would the box be without these. I still will want to sell the HD
> Theater 300.
> Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6541999>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAFYXPGVH4R7QRULMY3XR7TMJANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by gregort3** [comment]
Name and Address

Greg Wilm
710 North Post Oak Rd
Suite # 400
Houston Texas 77024 

If you like to buy the 3rd  HD300S Let me know , If not let me know.
Also Please let me know a Address to Send these to. 
If you want all 3 I will also Send a STP-HD200 with the order.

**Comment by weston1968** [comment]
Ok will get out a check for $175 tonight. Please let me know if this is ok

**Comment by gregort3** [comment]
This Deal is Good to Me

I will be sending you 3x HD300S and 1x STP-HD200
Please Let me know your Address

Thanks
Greg

**Comment by weston1968** [comment]
Lance Weston
9 Claudine Court
East Northport, NY
11731

**Comment by weston1968** [comment]
I cut the check.  It will arrive Aug 3.  Please look carefully.  It is
coming from TD bank and will look like junk mail.

thanks
Lance

[image: image.png]

On Thu, Jul 27, 2023 at 1:05 PM gregort3 ***@***.***> wrote:

> This Deal is Good to Me
>
> I will be sending you 3x HD300S and 1x STP-HD200
> Please Let me know your Address
>
> Thanks
> Greg
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6567806>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAEW36HAHWAMZYW72RLXSKNXHANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Reply by gregort3** [reply]
Weston

I just checked my mail this morning I have not received a check as yet. 
My mail usually comes between 11 and 12 pm , I will keep you posted after Lunch, I do have everything boxed and ready to send you.

I will give you a tracking number as well.

Thanks

greg

**Comment by eelpout** [comment]
So you sold all 3 hd300's?

**Comment by gregort3** [comment]
Yes I did sell all three

**Comment by weston1968** [comment]
Hi Greg,

You should have gotten the check today.  i may have looked like junk mail
from TDbank.  Please send me the tracking number.

Lance

On Thu, Jul 27, 2023 at 1:05 PM gregort3 ***@***.***> wrote:

> This Deal is Good to Me
>
> I will be sending you 3x HD300S and 1x STP-HD200
> Please Let me know your Address
>
> Thanks
> Greg
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6567806>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAEW36HAHWAMZYW72RLXSKNXHANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
Hi Greg,

Did the check arrive yesterday?

Lance

On Thu, Jul 27, 2023 at 1:05 PM gregort3 ***@***.***> wrote:

> This Deal is Good to Me
>
> I will be sending you 3x HD300S and 1x STP-HD200
> Please Let me know your Address
>
> Thanks
> Greg
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6567806>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAEW36HAHWAMZYW72RLXSKNXHANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Reply by gregort3** [reply]
I am sure you sent to 

Greg Wilm
710 North Post Oak Rd
Suite #400
Houston Texas 77024

**Comment by weston1968** [comment]
When my 3 arrive I will sell you one for the same deal

**Reply by gregort3** [reply]
Weston I just got you check in the mail.
A little later today I will be going to send the Equipment
I will send you a Tracking number a little later today

Thanks
Greg

**Reply by eelpout** [reply]
Cool. Let me know when they arrive and we'll work out the details.

**Comment by weston1968** [comment]
Sounds great, thank you!

**Comment by weston1968** [comment]
No worries, never meant to take yours.  Assumed you dropped the idea.

Lance

On Fri, Aug 4, 2023 at 3:14 PM eelpout ***@***.***> wrote:

> Cool. Let me know when they arrive and we'll work out the details.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6641724>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAHQU6WKSGPS3YZNZODXTVCZVANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Reply by gregort3** [reply]
Lance Tracking #UPS
1Z2E2A704254733695 
Signature Required for Package

**Reply by gregort3** [reply]
I will go to UPS and see if they can take off the signature

**Reply by gregort3** [reply]
Just doing to make sure would not get stolen with a porch pirate

**Reply by gregort3** [reply]
Weston

I went to UPS they said after Shipping I cannot change Authorize Signature on Package
They said they will try to Deliver 3 times. They also said you could have the package delivered to Local UPS and then could pick
it up from there. 

Can you change signature required for UPS?
If the sender has specified that an adult signature is required, you cannot authorize delivery release online on UPS. You can, however, redirect it to a UPS customer center. You could either have your package left at a UPS location, such as The UPS Store or a UPS Access Point® location or change your instructions.

Sorry about the problem , thought I was making it safter

Greg

**Comment by weston1968** [comment]
This is address of check

The checks usually arrive on the day specified which was the 3rd.  Have no
idea what happened.


OptionalNickname
Category
Automobile Loan Banking Deposit Cable/Satellite TV Cellular Charitable
Donations Childcare Consumer Loan Credit Card Education Entertainment
Fuel Garbage
& Recycle Healthcare Home Equity Household Insurance Lawn Care Line of
Credit Membership Fees Miscellaneous Mortgage Newspaper/Books Online/Internet
Service Parking Parts/Supplies Personal Care Pet Care Rent Student Loan
Taxes Telephone Transportation Uncategorized Utilities Vacation Add a new
category
About adding categories
Address 1
OptionalAddress 2
City
State
StateAA AE AK AL AP AR AS AZ CA CO CT DC DE FL FM GA GU HI IA ID IL IN KS KY
 LA MA MD ME MH MI MN MO MP MS MT NC ND NE NH NJ NM NV NY OH OK OR PA PR PW
RI SC SD TN TX UT VA VI VT WA WI WV WY
ZIP Code
-Optional four numbers of the ZIP Code
Phone Number

×Payment Details

   - FinishedPending
   - FinishedProcessing
   - NowProcessed

Greg Wilm

Check 995521 was mailed to Greg Wilm for receipt by Aug 3, 2023. Your check
may be cashed, and the money withdrawn, before, on, or after Aug 3, 2023.
Pay FromTD RELATIONSHIP CHECKING x7492Amount$175.00
CHECK
PROCESSEDAugustAug 3 (Estimated)press enter to open tooltip
ConfirmationV6XFZ-3HZ4D
Printthe payment details

On Fri, Aug 4, 2023 at 9:39 AM gregort3 ***@***.***> wrote:

> I am sure you sent to
>
> Greg Wilm
> 710 North Post Oak Rd
> Suite #400
> Houston Texas 77024
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6638806>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAASYBS4EN2A4HKPSVLXTT3QTANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
much thanks

On Sat, Aug 5, 2023 at 9:24 AM gregort3 ***@***.***> wrote:

> Lance Tracking #UPS
> 1Z2E2A704254733695
> Signature Required for Package
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6645661>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAGCTGWTBRCKUEYZW2LXTZCRFANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
Wish you had asked first before signature request, it will probably come
back to you.  I have a relative dying of cancer and all of my time is at
the hospital.  No way can I sign for the package.  Can you contact UPS and
remove that.

Lance

On Sat, Aug 5, 2023 at 9:24 AM gregort3 ***@***.***> wrote:

> Lance Tracking #UPS
> 1Z2E2A704254733695
> Signature Required for Package
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6645661>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAGCTGWTBRCKUEYZW2LXTZCRFANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
thanks

On Sat, Aug 5, 2023 at 9:42 AM gregort3 ***@***.***> wrote:

> I will go to UPS and see if they can take off the signature
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6645752>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAFWA2O5FJ6JIA7CJODXTZESXANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
No worries here.  I have packages worth thousands just dropped on the
porch, everybody does.   Live on a court, everyone has cameras with 3
retired cops and nearly everyone is armed.  I understand your concern.  But
for me, right now,  it just has the effect of not being able to receive the
package.

I had a neighborhood kid put one of the joke bombs on my porch.  I stepped
on it and it went off.  Just the type of thing I did as a kid, before
cameras.  Of course I had pictures of him from many angles.  Printed out
his picture.  I found him and was able to not laugh and seem stern.  I
showed him the picture and said I wanted to take him to his dad.  Kid
turned white, I let him go and told him to tell his friends that our block
was not the place to screw around.  Been five years, no problems.

Lance

On Sat, Aug 5, 2023 at 9:42 AM gregort3 ***@***.***> wrote:

> Just doing to make sure would not get stolen with a porch pirate
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6645755>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAG7HOEYX7VA76G2ZR3XTZEWBANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
I apparently have an old login and UPS will not let me login or reset my
login.  I can not arrange a different drop off without logging in and can
not wait a full day for delivery.  My days are at the hospital.  When it
comes back to you please reship without a signature requirement.

Lance


On Sat, Aug 5, 2023 at 11:40 AM gregort3 ***@***.***> wrote:

> Weston
>
> I went to UPS they said after Shipping I cannot change Authorize Signature
> on Package
> They said they will try to Deliver 3 times. They also said you could have
> the package delivered to Local UPS and then could pick
> it up from there.
>
> Can you change signature required for UPS?
> If the sender has specified that an adult signature is required, you
> cannot authorize delivery release online on UPS. You can, however, redirect
> it to a UPS customer center. You could either have your package left at a
> UPS location, such as The UPS Store or a UPS Access Point® location or
> change your instructions.
>
> Sorry about the problem , thought I was making it safter
>
> Greg
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6646313>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAFNDYZHDBKHFQ5YFPDXTZSO5ANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Reply by gregort3** [reply]
I have a suggestion , UPS should leave a message on your door  when package delivered , sign and say deliver to local UPS , then can pick up any time UPS is open.

**Comment by weston1968** [comment]
Finally got my UPS account straightened out.  I redirected to the local UPS
store for a cost of $6.  UPS says no guarantee that they can redirect.

On Sun, Aug 6, 2023 at 8:34 AM gregort3 ***@***.***> wrote:

> I have a suggestion , UPS should leave a message on your door when package
> delivered , sign and say deliver to local UPS , then can pick up any time
> UPS is open.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6650536>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAGFWDV7TQ7JAL7UM2LXT6FOLANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Reply by gregort3** [reply]
Sorry about all the problems, I just thought Signature would be a good thing , Little did I know LOL

**Comment by weston1968** [comment]
I probably can get tothe UPS store before closing  PayPal $60 for the HD300
and shipping and I will try to send it out tomorrow

**Reply by eelpout** [reply]
great. do you do PayPal?

**Reply by weston1968** [reply]
looks like github will not let me send you my email for paypal.  Just send a check for $60.  Include your address.

**Reply by eelpout** [reply]
The Weston address above, correct? OK, I'll get it out today. Address will be on the check.

**Comment by weston1968** [comment]
Yup. ***@***.***
Give me your address

**Reply by gregort3** [reply]
Weston so it looks like Equipment is in your Local UPS store.
Let me know when you have HD 300 s

**Comment by weston1968** [comment]
just picked up, all three working.

On Wed, Aug 9, 2023 at 4:21 PM gregort3 ***@***.***> wrote:

> Weston so it looks like Equipment is in your Local UPS store.
> Let me know when you have HD 300 s
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6684393>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAA2NKLT2GWK3T42GMDXUPWNBANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
Yup

**Comment by weston1968** [comment]
Thank you
I have now tested the HD200 as well as the 3 HD300's.  I do not see any
difference between HD200 and HD300 so I am happy.

Lance


On Wed, Aug 9, 2023 at 4:21 PM gregort3 ***@***.***> wrote:

> Weston so it looks like Equipment is in your Local UPS store.
> Let me know when you have HD 300 s
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6684393>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAA2NKLT2GWK3T42GMDXUPWNBANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Reply by gregort3** [reply]
Lance
I am glad you are happy with the HD300 and HD200 , HD300 in the specs was about 100MHZ faster than the HD200 if I remember correctly.
I gave up on the SageTV  because 2 to 3 times a year Sage Server OS would stop working , Most of the time it was a Microsoft Update that caused the issue. At this time I figured out how to VM Sage Server.  USB Uart update from Microsoft would stop the channels from getting changed.
I might have had better luck if I had used  Linux Server ,

**Reply by eelpout** [reply]
the HD300, has a newer Sigma chip and twice the memory. but I think if one doesn't have a lot of plugins and just runs the usual video containers, you don't notice (I still run an HD200 in a bedroom).

**Comment by weston1968** [comment]
Check did not arrive today.  Unit all boxed up and ready to go.
Lance

On Fri, Aug 11, 2023 at 11:52 AM eelpout ***@***.***> wrote:

> The Weston address above, correct? OK, I'll get it out today. Address will
> be on the check.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6703184>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RACZ7MNIXKMF4NWWJ6LXUZINDANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Reply by eelpout** [reply]
Cool. I don't think the checked was picked up until Friday. it'll take several days to cross the country. 

so great that because of GitHub and no private messaging we are living in 1967. ;)

**Comment by weston1968** [comment]
I kept win 7 and then refused updates.  The system has stayed stable.  The
computer is used only for Sage.

On Tue, Aug 15, 2023 at 9:56 AM gregort3 ***@***.***> wrote:

> Lance
> I am glad you are happy with the HD300 and HD200 , HD300 in the specs was
> about 100MHZ faster than the HD200 if I remember correctly.
> I gave up on the SageTV because 2 to 3 times a year Sage Server OS would
> stop working , Most of the time it was a Microsoft Update that caused the
> issue. At this time I figured out how to VM Sage Server. USB Uart update
> from Microsoft would stop the channels from getting changed.
> I might have had better luck if I had used Linux Server ,
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6730892>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAFV5AI3KWVP4ZQUQBLXVN5ZLANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Comment by weston1968** [comment]
Got check, package goes out in tomorrow's lunch time mailing.  Priority
mail.

Lance

On Tue, Aug 15, 2023 at 8:09 PM eelpout ***@***.***> wrote:

> the HD300, has a newer Sigma chip and twice the memory. but I think if one
> doesn't have a lot of plugins and just runs the usual video containers, you
> don't notice (I still run an HD200 in a bedroom).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6735391>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAHHC4V3SXAGV6QAOKDXVQFVDANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728

**Reply by eelpout** [reply]
thanks! :)

**Reply by eelpout** [reply]
Lance, package received. I have yet to plug it in but I'm sure it's fine. Thanks! :)

**Comment by weston1968** [comment]
I was wrong on the price of shipping.  $10.20 not $10.00.  tracking number
9505 5163 0287 3229 7031 37

Lance

On Thu, Aug 17, 2023 at 12:06 PM eelpout ***@***.***> wrote:

> thanks! :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/23#discussioncomment-6753039>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RACMIQUPCAPO3AWNW5LXVY6PFANCNFSM6AAAAAAYI3DSFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728
